src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx

```diff
@@ -77,137 +63,206 @@ helm upgrade --install newrelic-bundle newrelic/nri-bundle \
 To install the Kubernetes agent operator with the default configuration, run these commands:
 
 ```shell
-helm repo add newrelic-k8s https://newrelic.github.io/k8s-agents-operator
-helm upgrade --install k8s-agents-operator newrelic-k8s/k8s-agents-operator \
+helm repo add k8s-agents-operator https://newrelic.github.io/k8s-agents-operator
+helm upgrade --install k8s-agents-operator k8s-agents-operator/k8s-agents-operator \
     --namespace newrelic \
     --create-namespace \
     --set licenseKey=YOUR_NEW_RELIC_INGEST_LICENSE_KEY
 ```
 
 For a complete list of configuration options, see the [README](https://github.com/newrelic/k8s-agents-operator/tree/main/charts/k8s-agents-operator#values) chart.
 
-## Configuring a namespace to enable the operator [#configuring-namespace]
+## Monitored namespaces [#monitored-namespaces]
 
-Each namespace requires a two-step process to enable automatic instrumentation of applications deployed within it.
+For each namespace you want the operator to be instrumented, a secret will be replicated from the newrelic operator namespace.
```

Comment:
This line is a little confusing. What exactly is being replicated? "A secret ____ will be replicated from the newrelic operator namespace." Also, should newrelic operator be in the code format?

Reply:
We basically replicate the Kubernetes secret into a new namespace if the customer wants to instrument a pod that is in a new namespace that has no monitoring. No need for a code block. Perhaps the following wording is better:

A Kubernetes secret will automatically be replicated from the operator's namespace into all namespaces that a pod will be monitored in.